### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by foster999**
Add a pull request template, including a checklist to indicate what checks have been carried out prior to review.

### CODE REVIEW

**Comment by foster999** [comment]
@ONS-SST/cis-dev any thoughts on what else should be included here?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Overview of changes
+
+<!---
+Add any useful details about your changes here.
+-->
+
+## Post-review checklist
+
+As I merege these changes, I have:
```
Comment (jonathonmellor) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: Typo: merege

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- [ ] Updated the related Jira ticket.
```
Comment (jonathonmellor) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: Added the time for development and review as well?

Comment (jonathonmellor) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: + moved it to done explicitly?

**Comment by jonathonmellor** [comment]
I'm happy to approve when comments are addressed. Would be good to have another member of the team double check though as we will all need to follow the standard.

**Review by jonathonmellor — APPROVED** [review]

**Pull request merged**